lib/components/FieldMessage/FieldMessage.treat.ts

```diff
@@ -0,0 +1,33 @@
+import { style } from 'sku/treat';
+
+export const root = style(
+  {
+    justifyContent: 'flex-end',
+  },
+  'root',
+);
+
+export const grow = style(
+  {
+    flexGrow: 1,
+  },
+  'grow',
```

Comment:
If you used `css` instead we wouldn't need the debug name here. Personally, think that would be cleaner.

Reply:
As discussed, gonna leave this it as-is until we optimise the output of themed `css` blocks.